infra/main.bicep

```diff
@@ -53,6 +53,27 @@ param machineLearningName string = '${environmentName}-aml-${resourceToken}'
 @description('Id of the user or app to assign application roles')
 param principalId string = ''
 
+@description('Flag to deploy resources in isolated network. Manual setup is required to access the deployed resources.')
+param DeployResourcesWithIsolatedNetwork bool
+
+@description('Name of the virtual network')
```

Comment:
does this need to be a param?
We don't set it, so why not leave it as a variable, rather than a parameter?